llvm/include/llvm/IR/InstrTypes.h

```diff
@@ -311,6 +312,31 @@ class BinaryOperator : public Instruction {
     return BO;
   }
 
+  static BinaryOperator *
+  CreateWithFMF(BinaryOps Opc, Value *V1, Value *V2, FastMathFlags FMF,
+                const Twine &Name = "", Instruction *InsertBefore = nullptr) {
+    BinaryOperator *BO = Create(Opc, V1, V2, Name, InsertBefore);
```

Comment:
If you added a new operand/overload to Create, can you avoid the new include?

Reply:
I don't think so. The include is needed because I'm passing FastMathFlags by value. It gets passed as a single integer, so it doesn't really make sense to pass by pointer.